### PR TITLE
[FIXED] Footer Github Accelerator Image

### LINKED ISSUE #6658: Footer GitHub accelerator image pixelates on larger screens
### Description

A higher quality image can be added and a `bg-contain` classname for smaller devices?

image

image

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by dhawansolanki**
Closes #6658 

## Fixes Issue

Closes #6658 

## Changes proposed

1. Changed Image Background Size from cover to contain.
2. Added background Color to Image.

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots
Old Screenshot : 
236642719-fcc5a489-5d21-4906-972e-24e064ca072f

New Screenshot after Changes : 
image

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6831"></a>

### CODE REVIEW

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
From the issue...

> A higher quality image can be added and a bg-contain classname for smaller devices?

Thank you for the contribution, but I am confused, the issue was about replacing the image with a higher quality version, but the image is still the same and there have been code changes?

File: components/GitHubAccelerator.js [hunk 1]
```diff
@@ -8,11 +8,12 @@ export default function GitHubAccelerator() {
     <div className="mb-12">
       <div>
         <Image
-          className="h-32 w-full object-cover lg:h-48"
+          className="h-32 w-full lg:h-48"
           src="https://user-images.githubusercontent.com/624760/235968674-01cc3149-f9c3-48e2-9dc5-677789de8456.png"
           alt="GitHub Accelerator banner"
           width={600}
           height={200}
+          style={{"object-fit": "contain","background": "#241e44"}}
```
Comment (eddiejaoude) [inline components/GitHubAccelerator.js:16]: This seems to give an error

```
Warning: Unsupported style property object-fit. Did you mean objectFit?
```

Comment (ChinmayMhatre) [reply components/GitHubAccelerator.js:16]: Object fit can be given using the tailwind classname will add a comment.

File: components/GitHubAccelerator.js [hunk 2]
```diff
@@ -8,11 +8,12 @@ export default function GitHubAccelerator() {
     <div className="mb-12">
       <div>
         <Image
-          className="h-32 w-full object-cover lg:h-48"
+          className="h-32 w-full lg:h-48"
```
Comment (ChinmayMhatre) [inline components/GitHubAccelerator.js:11]: Just add the contain property here.

```suggestion
          className="h-32 w-full lg:h-48 object-contain"
```

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]

**Comment by ChinmayMhatre** [comment]
@dhawansolanki as Eddie suggested you do need to add a higher quality image as mentioned in the issue.

**Comment by eddiejaoude** [comment]
Thank you for your contribution. It has been 2 weeks with no activity, so I will close this pull request and unassign the issue as it is blocking others from moving this forwards.

For more information please read our contributing guidelines https://github.com/EddieHubCommunity/LinkFree/blob/main/CONTRIBUTING.md

**Pull request closed without merging**